### PR TITLE
Update support for currently supported ruby versions

### PR DESCRIPTION
**Pull request by j15e**
I think we should merge this then merge #27 and release a new major version (no new release with 1.9 support).

cc @aleksandrs-ledovskis @exoszajzbuk

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,11 @@ bundler_args: --without debug release
 before_install:
   - gem install bundler
 rvm:
-  - 1.9.3
-  - 2.0.0
+  - 2.1.9
```
Comment (aleksandrs-ledovskis) [inline .travis.yml]: Maybe go extra step and give ["2.1.10"](https://www.ruby-lang.org/en/news/2016/04/01/ruby-2-1-10-released/) a chance?

File: capistrano-s3.gemspec [hunk 2]
```diff
@@ -18,6 +18,9 @@ Gem::Specification.new do |s|
   s.cert_chain  = ['certs/aleksandrs-ledovskis.pem']
   s.signing_key = File.expand_path("~/.ssh/gem-private_key.pem") if $0 =~ /gem\z/
 
+  # Min rubies
+  s.required_ruby_version = ">= 2.0"
```
Comment (aleksandrs-ledovskis) [inline capistrano-s3.gemspec]: Pro/Cons of pushing the bar all the way to 2.1?

+ 2.0 has [been EOL'ed](https://www.ruby-lang.org/en/news/2016/02/24/support-plan-of-ruby-2-0-0-and-2-1/) for a good 6 months
+ Having 2.0 as a lowest acceptable version necessitates its inclusion in CI matrix

Comment (j15e) [inline capistrano-s3.gemspec]: Im okay with that, will update

File: .travis.yml (hunk 1 above)
Comment (j15e) [inline .travis.yml]: Yeah I missed it because `.10` is listed way after `.9` on http://rubies.travis-ci.org/

Will fix

**Comment by exoszajzbuk** [comment]
For a new release I have another feature in https://github.com/team-supercharge/capistrano-s3/commit/8a7a9d0eacc43900575a3aebc3955acbca4dad4d which will solve #3

**Pull request merged**